ghcide/ghcide.cabal

```diff
@@ -219,6 +220,7 @@ library
                 -Wincomplete-uni-patterns
                 -Wno-unticked-promoted-constructors
                 -fno-ignore-asserts
+                -j12
```

Comment:
this is bad for GHC <9.2

Reply:
Yes, these were commited by mistake, I will remove them.